pyscal/factory.py

```diff
@@ -931,7 +986,7 @@ def create_wateroilgas_list(
             PyscalList, consisting of WaterOilGas objects
         """
         wogl = PyscalList()
-        for (row_idx, params) in relperm_params_df.sort_values("SATNUM").iterrows():
+        for row_idx, params in relperm_params_df.sort_values("SATNUM").iterrows():
```

Comment:
This is in master already, I guess you rebased off an outdated master branch (?)

Reply:
It seems my branch did not include the latest commit ( #393) by @alifbe.

Reply:
Yes, if you just squash on your local "master", then this will happen. You can do `git rebase -i origin/master` (if origin is your origin), or you checkout master and do a git pull there, then switch back to your branch and do `git rebase -i master`.

Reply:
You can (in this case equivalently) also ensure you select "squash" when clicking the merge button in the github gui, then Github will do the same for you (avoiding having the merge commit in the code history).

Reply:
ok, so basically I should click the "Squash and merge" button at the bottom here?

Reply:
My bad, clicking "squash and merge" button didnt avoid merge commit in the history... is there any way to fix this or its too late now?